releng/org.eclipse.xtext.contributor/Xtext.setup

```diff
@@ -1342,6 +1270,233 @@
+          <repository
+              url="${p2.antlr-gen}"/>
+          <repository
+              url="${p2.gef}"/>
+          <repository
+              url="${p2.m2e}/1.8"/>
+          <repository
+              url="${p2.swtbot}"/>
+          <repository
+              url="${p2.xtext.orbit}"/>
+          <repository
+              url="${p2.emf}"/>
+          <repository
+              url="${p2.draw2d}"/>
+          <repository
+              url="${p2.xpand}"/>
+          <repository
+              url="${p2.buildship}"/>
+          <repository
+              url="${p2.webtools}"/>
+        </repositoryList>
+        <repositoryList
+            name="Photon - Staging">
+          <repository
+              url="${p2.xtext}"/>
+          <repository
+              url="${p2.mwe2}"/>
+          <repository
+              url="${p2.antlr-gen}"/>
+          <repository
+              url="${p2.gef}"/>
+          <repository
+              url="${p2.m2e}/1.8"/>
+          <repository
+              url="${p2.swtbot}"/>
+          <repository
+              url="${p2.xtext.orbit}"/>
+          <repository
+              url="${p2.emf}"/>
+          <repository
+              url="${p2.draw2d}"/>
+          <repository
+              url="http://download.eclipse.org/staging/photon/"/>
+          <repository
+              url="${p2.xpand}"/>
+          <repository
+              url="${p2.buildship}"/>
+          <repository
+              url="${p2.webtools}"/>
+        </repositoryList>
+        <repositoryList
+            name="Oxygen">
+          <repository
+              url="${p2.xtext}"/>
+          <repository
+              url="${p2.mwe2}"/>
+          <repository
+              url="${p2.antlr-gen}"/>
+          <repository
+              url="${p2.gef}"/>
+          <repository
+              url="${p2.m2e}/1.8"/>
+          <repository
+              url="${p2.swtbot}"/>
+          <repository
+              url="${p2.xtext.orbit}"/>
+          <repository
+              url="${p2.emf}"/>
+          <repository
+              url="${p2.draw2d}"/>
+          <repository
+              url="${p2.xpand}"/>
+          <repository
+              url="${p2.buildship}"/>
+          <repository
+              url="${p2.webtools}"/>
+        </repositoryList>
+        <repositoryList
+            name="Oxygen - Staging">
+          <repository
+              url="${p2.xtext}"/>
+          <repository
+              url="${p2.mwe2}"/>
+          <repository
+              url="${p2.antlr-gen}"/>
+          <repository
+              url="${p2.gef}"/>
+          <repository
+              url="${p2.m2e}/1.8"/>
+          <repository
+              url="${p2.swtbot}"/>
+          <repository
+              url="${p2.xtext.orbit}"/>
+          <repository
+              url="${p2.emf}"/>
+          <repository
+              url="${p2.draw2d}"/>
+          <repository
+              url="http://download.eclipse.org/staging/oxygen/"/>
```

Comment:
sicher dass das oxygen 2 enbthält. für mich sieht das nach nur oygen aus (ist auch in meiner target leider noch so drinne)

Reply:
You would have to choose "Oxygen" instead of "Oxygen Staging". The "Staging" platforms only make sense when you really want to use staging instead of release.

Reply:
No is there a oxygen staging for .2??